spec/features/idv/doc_auth/verify_info_step_spec.rb

```diff
@@ -333,4 +333,316 @@
       expect(page).to have_current_path(idv_cancel_path(step: :gpo))
     end
   end
+
+  context 'with split doc auth', allow_browser_log: true do
```

Comment:
@AShukla-GSA - can you please share the value of adding these tests in the verify info step?